db/re/item_db_equip.yml

```diff
@@ -105441,6 +105443,7 @@ Body:
       NoAuction: true
     Script: |
       bonus bMaxHP,100;
+      bonus bMatk,15;
```

Comment:
The matk is already written in the MagicAttack field
```suggestion
```

Reply:
Thanks